src/Controller/Project/ProcesVerbalController.php

```diff
@@ -142,7 +142,7 @@ public function rediger(Request $request, Etude $etude, $type, EtudePermissionCh
 
         if (!$procesverbal = $etude->getDoc($type)) {
             $procesverbal = new ProcesVerbal();
-            if ('PVR' == strtoupper($type)) {
+            if ('PVRF' == strtoupper($type)) {
```

Comment:
Suggestion d'amélioration (le problème date d'avant ta modification, mais quitte à modifier le code à coté, on peut aussi traiter ça), utiliser les constantes du `TraitementController` au lieu des chaînes de caractères codées en dur dans les différents fichiers.

Exemple:

```suggestion
            if (App\Controller\Publish\TraitementController::DOCTYPE_PROCES_VERBAL_FINAL == strtoupper($type)) {
```